tests/tests_integration/test_api/test_functions.py

```diff
@@ -16,3 +16,13 @@ def test_retrieve_unknown_raises_error(self, cognite_client: CogniteClient):
     def test_retrieve_unknown_ignore_unknowns(self, cognite_client: CogniteClient):
         res = cognite_client.functions.retrieve_multiple(external_ids=["this does not exist"], ignore_unknown_ids=True)
         assert len(res) == 0
+
+    def test_function_list_schedules_unlimited(self, cognite_client: CogniteClient):
+        expected_unique_schedules = 100
```

Comment:
Let's pick a lower number (integration tests put quite a lot of pressure on CDF already). Maybe 5?